### PR TITLE
Fix IO capture for calculating RID.

### PR DESCRIPTION
**Pull request by russellhadley**
Privious change broke this functionality.

### CODE REVIEW

**Comment by russellhadley** [comment]
@AndyAyersMS @pgavlin I broke the RID setting functionality in my last change.  This fixes it.

File: src/jit-diff/jit-diff.cs [hunk 1]
```diff
@@ -551,15 +551,24 @@ public static int Main(string[] args)
...
             try 
             {
                 Command command =  Command.Create(name, commandArgs);
 
-                // Wireup stdout/stderr so we can see outout.
-                command.ForwardStdOut();
-                command.ForwardStdErr();
+                if (capture)
+                {
+                    // Capture stdout/stderr for consuption within tool.
```
Comment (AndyAyersMS) [inline src/jit-diff/jit-diff.cs]: Typo in comment 'consuption'

File: src/jit-diff/jit-diff.cs [hunk 2]
```diff
@@ -551,15 +551,24 @@ public static int Main(string[] args)
...
+                    command.CaptureStdOut();
+                    command.CaptureStdErr();
+                }
+                else
+                {
+                    // Wireup stdout/stderr so we can see outout.
+                    command.ForwardStdOut();
```
Comment (AndyAyersMS) [inline src/jit-diff/jit-diff.cs]: Typo 'outout'

**Comment by AndyAyersMS** [comment]
Changes LGTM. A couple typos in comments.

**Pull request merged**